lib/Gemfile

```diff
@@ -42,7 +42,12 @@ gem "rufus-lru",            "~>1.0.3",        :require => false
 gem "uuidtools",            "~>2.1.3",        :require => false
 gem "sass",                 "3.1.20",         :require => false
 gem "trollop",              "~>2.0",          :require => false
-gem "psych",                "~>2.0.12"
+
+# JRuby has its own version of psych
+if RUBY_PLATFORM != 'java'
+  gem "psych",              "~>2.0.12"
+end
```

Comment:
@kbrock sorry, I've been dragging my feet on that.  I'll try to get it pushed out today.